### ISSUE TITLE
Unable to solve local type declared and used in method

### ISSUE THREAD

**Issue by mlangkabel** [javaparser/javasymbolsolver#278]
In the following code snippet the type of `lc` cannot be solved. It seems that the `AbstractMethodLikeDeclarationContext` does not handle local types.

```java
class Main {
    public void foo() {
        class LocalClass {
        }
        LocalClass lc;
    }
}
```

**Comment by mlangkabel** [comment]
Ok, I have got a fix for this. Just have to implement a test case and I can push so that we can merge it.

**Comment by ftomassetti** [comment]
great!

**Comment by mlangkabel** [comment]
fixed with merge of pull request #279
